src/main.ts

```diff
@@ -0,0 +1,89 @@
+import { Command, Helper } from 'dojo-cli/interfaces';
+import { Argv } from 'yargs';
+import * as fs from 'fs';
+import * as path from 'path';
+const DtsCreator: any = require('typed-css-modules');
+const read: any = require('read-file-stdin');
+const globby: any = require('globby');
+const async: any = require('async');
+const mkdirp: any = require('mkdirp');
+const chalk: any = require('chalk');
+
+interface TypingsArgs extends Argv {
+	out: string;
+}
+
+function processFile(args: TypingsArgs, creator: any, file: string, done: Function): void {
```

Comment:
It would be cleaner if you could move this function to it's own file, then the main file that returns the command can be more streamlined.